entity-framework/dotnet-data/index.yml

```diff
@@ -0,0 +1,216 @@
+### YamlMime:Hub
+
+title: .NET data
+summary: .NET is the open source and cross-platform for integrating data, data services and data backends with your applications.
+brand: dotnet
+
+metadata:
+  title: .NET data documentation
```

Comment:
Can we style it *.NET Data*? (everywhere) This just makes me ask, "What's data documentation?"